client/js/components/layout/index.jsx

```diff
@@ -6,6 +6,10 @@ export default class Index extends React.Component {
     children: React.PropTypes.node,
   };
 
+  static defaultProps = {
+    children: '',
+  }
+
```

Comment:
Sets the default value for that prop to an empty string. The linter requires a default value if the propType is not required.